sbnanaobj/StandardRecord/SRNuID.h

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRNuID.h
+// \brief   SRNuID object for Neutrino ID score features (MVA inputs). This SR code copied/based on other SR objects.
+// \author  $Author: howard@fnal.gov
+////////////////////////////////////////////////////////////////////////
+#ifndef SRNUID_H
+#define SRNUID_H
+
+namespace caf
+{
+  /// Slice Neutrino ID Score features (MVA inputs)
+  class SRNuID
+  {
+  public:
+    SRNuID();
```

Comment:
I confirm that, beyond the choice of syntax, the recommendation is the one @brucehoward-physics described.
We do have a guideline ([`[CF-060]`](https://github.com/SBNSoftware/SBNSoftware.github.io/blob/master/sbn/codingconv/CodingConventionsExplained.md#CF060)) _suggesting_ the use of braces. That is intended as a tie-breaker in case your mind is stuck on a tie.

Some differences between `=` and `{}` syntax (whose nickname "universal initialization" betrays its ambition) for simple type data member initialization are that the latter will complain about "narrowing" (for example if you try to assign a `double` value to a `float` variable), and that it allows explicit default initialization, for example:
```cpp
  unsigned short int fNADC {};
  short int fADC[16] {};
```
will initialize `fNADC` to `0` and all `fADC` to `0` too (with no initializer, both are undefined). You can obtain the same effect with `= 0` and `= { 0 }` respectively.
Not much difference for simple types, then. When object data members are involved, braces are a superior option (but usually equivalent to parentheses, except for noticeable exceptions like [`std::vector<int>`](https://alexsm.com/cpp-std-vector-braced-init) and for its role in [the most vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse)).
I try to use braces mostly because since it's newer syntax it makes me look smarter. But my official excuse is that it's more uniform so I can usually manage to have only one type of syntax (available: `int a = 0;`, `int a (0);`, `int a { 0 };`, `int a = { 0 };`; so yeah, [`{}` is the new universal standard](https://xkcd.com/927)).

Reply:
Okay - have pushed commits which add a central `kSignalingNaN` and applies it to SRSlice and SRNuID (I didn't touch the other StandardRecord objects).

I have it set up to use braces. There are some items which are not seemingly not being forced to initialize in SRSlice. Shall I add an empty `{}` here too?